src/main/kotlin/org/rucca/cheese/model/PatchTeamMemberRequestDTO.kt

```
@@ -0,0 +1,13 @@
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import io.swagger.v3.oas.annotations.media.Schema
+import javax.validation.Valid
+
+/** @param role */
+data class PatchTeamMemberRequestDTO(
+    @field:Valid
+    @Schema(example = "null", description = "")
+    @get:JsonProperty("role")
+    val role: TeamMemberRoleTypeDTO? = null
+) {}
```

detekt

[warning] The class or object PatchTeamMemberRequestDTO is empty.

(detekt.empty-blocks.EmptyClassBlock)